lock.go

```diff
@@ -29,6 +29,12 @@ const BACKUP_LOCK LockType = 1
 const RESTORE_LOCK LockType = 1
 const DELETION_LOCK LockType = 2
 
+type Operation string
+
+const CREATERESTORE Operation = "create/restore"
+const DELETE Operation = "delete"
+const UNDEFINED Operation = "undefined"
```

Comment:
The names are a bit ugly. Do we have better choices?
```suggestion
const (
  CREATERESTORE Operation = "create/restore"
  DELETE Operation = "delete"
  UNDEFINED Operation = "undefined"
)
```

Reply:
Why not separate CREATE and RESTORE

Reply:
because lock type CREATE and RESTORE are the same, they are all "1" since they have the same priority.
Thus we can't distinguish what operation it is by only checking the lock

Reply:
I see. Thank you.

Reply:
Can we add a prefix to the constants?

Reply:
Fixed. Thanks